scvi/model/_scvi.py

```diff
@@ -304,12 +285,21 @@ def to_latent_mode(
         """
         # TODO(adamgayoso): Add support for other latent modes, including a mode
         # in which no data is minified.
-        # This validates and sets a new adata manager
-        self.adata = self._get_reduced_adata(mode)
+        if self.module.use_observed_lib_size is False:
+            raise ValueError(
+                "Latent mode not supported when use_observed_lib_size is False"
+            )
+        reduced_adata = get_reduced_adata(self.adata, mode)
         if mode == _SCVI_LATENT_MODE:
-            self.adata.obsm[_SCVI_LATENT_QZM] = self.adata.obsm[use_latent_qzm_key]
-            self.adata.obsm[_SCVI_LATENT_QZV] = self.adata.obsm[use_latent_qzv_key]
+            reduced_adata.obsm[_SCVI_LATENT_QZM] = self.adata.obsm[use_latent_qzm_key]
+            reduced_adata.obsm[_SCVI_LATENT_QZV] = self.adata.obsm[use_latent_qzv_key]
+            counts = self.adata.X if layer is None else self.adata.layers[layer]
```

Comment:
this should get the adata_manager associated with adata and pull from the registry using `REGISTRY_KEYS.X_KEY`, which would then allow us to pull out the layer argument

```
manager = self.get_anndata_manager(adata)
manager.get_from_registry(..)
```